src/routes/safe/components/Apps/index.jsx

```diff
@@ -0,0 +1,210 @@
+// @flow
+import React, { useState, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+
+import ButtonLink from '../../../../components/layout/ButtonLink'
```

Comment:
I think we have absolute routes configured, coudl you please use one here to avoid ../../../../../../ ?

Reply:
done